core/src/main/java/org/everit/json/schema/internal/TimeFormatValidator.java

```diff
@@ -1,16 +1,15 @@
 package org.everit.json.schema.internal;
 
-import com.google.common.collect.ImmutableList;
-
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
+import java.util.Arrays;
 import java.util.List;
 
 /**
  * Implementation of the "time" format value.
  */
 public class TimeFormatValidator extends TemporalFormatValidator {
-    private static final List<String> FORMATS_ACCEPTED = ImmutableList.of(
+    private static final List<String> FORMATS_ACCEPTED = Arrays.asList(
```

Comment:
it should be `Collections.singletonList(...`

Reply:
```java
private static final List<String> FORMATS_ACCEPTED = Arrays.asList(
            "HH:mm:ssZ", "HH:mm:ss.[0-9]{1,9}Z, HH:mm:ss[+-]HH:mm, HH:mm:ss.[0-9]{1,9}[+-]HH:mm"
    );
```
The array has more than one elements.

Reply:
This logic (Collections.unmodifiableList) moved to TemporalFormatValidator:

```java
    TemporalFormatValidator(DateTimeFormatter formatter, List<String> formatsAccepted) {
        this.formatter = requireNonNull(formatter, "formatter cannot be null");
        this.formatsAccepted = Collections.unmodifiableList(formatsAccepted);
    }
```

Reply:
:+1: